test/context.test.ts

```diff
@@ -96,6 +96,17 @@ describe('Clova Skill Client Context: LaunchRequest', () => {
     });
   });
 
+  it('should set reprompt for response object', () => {
+    const speechInfo: Clova.SpeechInfoObject[] = [SpeechBuilder.createSpeechText('こんにちは')];
```

Comment:
For `SimpleSpeech` speechObject, would you change speechInfo to:
```js
const speechInfo: Clova.SpeechInfoObject = SpeechBuilder.createSpeechText(...);
````

Or you may also keep speechInfo as array but use `SpeechList` type for speechObject like:
```js
const speechObject: Clova.OutputSpeechList = {
  type: 'SpeechList',
  values: speechInfo,
};
```